.prettierrc

```diff
@@ -0,0 +1,16 @@
+{
+  "overrides": [
```

Comment:
Why do we remove prettier options for solidity?

Reply:
since we use only solhint
prettier creates some bad outputs.